examples/chart/teleport-auto-trustedcluster/Chart.yaml

```diff
@@ -1,8 +1,9 @@
 name: teleport-auto-trustedcluster
 apiVersion: v2
-version: 0.0.8
+version: 0.0.9
 appVersion: "6"
-description: Teleport trusted cluster installation which automatically joins itself back to the provided root cluster.
+description: "[deprecated] Teleport trusted cluster installation which automatically joins itself back to the provided root cluster."
```

Comment:
mention which chart to use instead

Reply:
We don't have a direct replacement for `teleport-auto-trustedcluster` unfortunately, and creating one isn't in the scope of #5582.